support/modules-artifact-gen/single-file-artifact-gen

```diff
@@ -178,6 +178,30 @@ else
   stat -c %a "${file}" > $permissions_file
 fi
 
+# Check the the passthrough_args and potentially modify them
+# to avoid conflicts or to let them override the already args
+# provided to mender-artifact
+passthrough_args_modified=" "
+parse=$(xargs -n 2 printf "%s %s\n" <<< $passthrough_args)
+while read -r flag arg; do
+  case $flag in
+    -T | --type)
+      echo "Error: Conflicting flag '$flag'. Already specified by the script."
```

Comment:
The point is not that it is already specified (after others like `-o` are also specified), the point is that it is reserved for the script to use (iow it is the script who _decides_ the payload type)